hazelcast-spring/src/main/resources/hazelcast-spring-5.3.xsd

```diff
@@ -2819,6 +2821,14 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:complexType name="alto-socket">
```

Comment:
What do you think about adding some documentation for these types? Some types in this file have it, some don't but I believe it would be good to have them

Reply:
We probably will deprecate this in next major upgrade so  don't think I will.